### DISCUSSION TITLE
app-release.aab

### DISCUSSION THREAD

**Discussion by aminmooridev** [quasarframework/quasar#12734] [CLI - Capacitor / Cordova modes]
Hi. I have a problem. when I use quasar build -m cordova -T android. qausar generate .aab file but I want apk. How can I generate apk?

**Comment by MilosPaunovic** [comment]
`quasar build -m android -- -- --packageType=apk`

Read through this [https://github.com/quasarframework/quasar/issues/9631](https://github.com/quasarframework/quasar/issues/9631) and check this one out [https://cordova.apache.org/docs/en/10.x/guide/platforms/android/](https://cordova.apache.org/docs/en/10.x/guide/platforms/android/)

**Reply by aminmooridev** [reply]
Wow. Thank you so much. it's working. but I have new problem
How can I fix this?


![photo_2022-03-10_23-39-14](https://user-images.githubusercontent.com/58544408/157746394-7244ce73-f790-4572-944a-c30b95304090.jpg)

![photo_2022-03-10_23-50-54](https://user-images.githubusercontent.com/58544408/157747996-af0ea419-0d31-4cb2-bcf1-165622788cca.jpg)

**Reply by MilosPaunovic** [reply]
This is now a separate issue and does not seem to be related with original question but to something in your code.

**Reply by aminmooridev** [reply]
Can you check it. please?

**Reply by BharatheshJaishankar** [reply]
Bro its because the apk is not signed.
Create a keystore and then sign it with jarsigner

**Reply by dperezch** [reply]
@aminmooridev you fix it? i have the same problem, the apk is signed and renamed but won't install

**Reply by aminmooridev** [reply]
@davidperez-curso yes. I fixed it. I use the capacitor https://quasar.dev/quasar-cli-vite/developing-capacitor-apps/preparation

**Reply by dperezch** [reply]
@aminmooridev thanks, i find other solution, *npm uninstall -g cordova* and *npm install -g cordova@10.0.0* ,  this version generate an apk file instead of aab file.
